website/src/blogs/bare-metal-to-production-stateful-workloads-openebs-and-rancher-2-x.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Bare metal to production stateful workloads- OpenEBS and Rancher 2.x
+author: Dave Cook
+author_info: Building Cooby Cloud at Hetzner
+date: 18-12-2018
+tags: OpenEBS, Stateful Workloads, Bare metal, Docker
+excerpt: Besides having a really cool name, Montel Intergalactic is also a cool company to work with. I met one of their engineers in an OpenEBS Slack forum and explained to him what I was up to
+---
+
+Besides having a really cool name, [Montel Intergalactic](https://www.montel.fi/index.en.html) is also a cool company to work with. I met one of their engineers in an OpenEBS Slack forum and explained to him what I was up to. The next thing was that within a week we had everything setup at Hetzner. All Montel wanted in return was this humble blog post. A note to others in the space, THIS is how you build customer loyalty and a simple gesture like this can go a long way.
+
+Needless to say when we decide on our next vendor these guys will be at the top of the list. In fact, they’ll probably be the only ones on that list (next to [OpenEBS](https://www.openebs.io/?__hstc=216392137.052f484bfa6105a863fd21af0f05de61.1579868893254.1579868893254.1579868893254.1&amp;__hssc=216392137.1.1579868893255&amp;__hsfp=3765904294) of course!).
+
+This blog will outline the detailed steps we took to achieve our goal of setting up a bare-metal Kubernetes cluster at Hetzner from scratch using Rancher 2.x, Hetzner node driver, OpenEBS, Cert-Manager, Nginx Ingress loadbalancing and the deployment of the Tutem — Hello World app to demonstrate loadbalancing and Let’s Encrypt.
+
+**NOTE:** All nodes are created with Hetzner CX21 type servers. Our root domain is `cooby.tech` and an `A` record called `rancher.cooby.tech` points to the instance IP.
+
+### Initial Setup
+
+We’ll be creating 4 Hetzner cluster servers:
+
+1. `front-1` nginx-ingress loadbalancing (type: worker)
+2. `control-1` control plane (type: etcd, control)
+3. `worker-1` first worker (type: worker)
+4. `worker-2` second worker (type: worker)
+
+- Spin up a Hetzner CX21 instance with a Ubuntu 18.04 image
+- Generate a keypair and make sure you can ssh into the instance
+- Install docker and docker-compose from your local machine:
+
+    curl https://gist.githubusercontent.com/gridworkz/d78c290c4e6fd7753dc21bb50601745a/raw/625fc3b7ddb7a654fe379f67d0bb9c6ac5b9413a/InstallDockerCompose | ssh -i ~/.ssh/<privkey> root@rancher.cooby.tech
+    “/bin/bash -s”
+
+- Login to the instance and add a docker-compose.yml file to run Rancher:
+
+![Login to instance](/public/images/blog/login-to-instance.png)
+- Run: 
+
+    docker-compose up -d
+
+- `Install Hetzner node-driver from:` [https://mxschmitt.github.io](https://mxschmitt.github.io/ui-driver-hetzner/)
+- `Create a node-template for frontend`
+- `Add label loadbalancer=true`
+- `Remove iscsi driver with cloud-init:`
+
+![Remove iscsi driver](/public/images/blog/remove-iscsi-driver.png)
```

Comment:
same as above

Reply:
no need to remove `public,` 
its coming file
![Screenshot from 2021-07-07 16-38-18](https://user-images.githubusercontent.com/10995416/124749443-dd15e500-df41-11eb-9a47-9f7d66084ad2.png)